packages/extension-typography/src/typography.ts

```diff
@@ -1,5 +1,29 @@
 import { Extension, textInputRule } from '@tiptap/core'
 
+export interface TypographyOptions {
+  emDash: false,
+  ellipsis: false,
+  openDoubleQuote: false,
+  closeDoubleQuote: false,
+  openSingleQuote: false,
+  closeSingleQuote: false,
+  leftArrow: false,
+  rightArrow: false,
+  copyright: false,
+  trademark: false,
+  registeredTrademark: false,
+  oneHalf: false,
+  plusMinus: false,
+  notEqual: false,
+  laquo: false,
+  raquo: false,
+  multiplication: false,
+  superscriptTwo: false,
+  superscriptThree: false,
+  oneQuarter: false,
+  threeQuarters: false,
+}
+
```

Comment:
These should be `boolean`, right?

Reply:
I followed the same approach used in `StarterKit`, where all rules are included by default and you can only disable the ones you don't want.

I can change this to boolean if you think that makes more sense. Let me know, and I'll make the change.